packages/core/src/database/find-many.ts

```diff
@@ -15,11 +14,8 @@ export const buildFindMany = <Schema extends SchemaLike, ReturnType extends Sche
   const isKeyOfSchema = isKeyOf(schema);
   const connectKeyValueWithEqualSign = (data: Partial<Schema>) =>
     Object.entries(data)
-      .map(
-        ([key, value]) =>
-          isKeyOfSchema(key) && sql`${fields[key]}=${convertToPrimitiveOrSql(key, value)}`
-      )
-      .filter((value): value is Truthy<typeof value> => notFalsy(value));
+      .filter((entry): entry is [keyof Schema & string, any] => isKeyOfSchema(entry[0]))
```

Comment:
[key] => isKeyOfSchema(key) ?

Reply:
Cannot 
<img width="982" alt="image" src="https://user-images.githubusercontent.com/10594507/157407135-705e4bcb-7945-4ea0-bb59-a837f81bd96d.png">
<img width="1405" alt="image" src="https://user-images.githubusercontent.com/10594507/157407697-9f0763c2-30ee-43c5-b93c-749c451135f5.png">

Because we need `(entry): entry is [keyof Schema & string, any]` for next line to infer `entry[0]` type as `keyof Schema & string`

You can have a try. Maybe there is a better way.